README.md

```diff
@@ -69,7 +68,48 @@ postgresql://postgres:password@localhost:5432/classroom
 
 </details>
 
-### Get the code
+### Setup Instructions
+
+<details>
+<summary>Codespaces</summary>
+
+1. Create a fork of the repository
+
+2. Open the Codespace by navigatig to your fork > click 'Code' > click the 'Codespaces' tab > click 'Open with Codespaces'
+
+3. After everything loads you should be able to see 3 URLs in your terminal 1. Mock user data URL, 2. Homepage URL and 3. Callback URL
+
+4. Update the MOCK_USER_DATA_URL and the NEXTAUTH_URL(Homepage URL) variables in the .env file with the respective URLs
+
+5. Setup an OAuth app via GitHub (you will add the client id and client secret to the .env file, see the following [documentation to setup an OAuth app](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app)) NOTE: use the Homepgae URL and the Callback URL to setup the app
+
+6. Update the .env file with the client id and client secret from the OAauth app
+
+7. Install Postgresql using the following commands
+
+   1. sudo apt update
+   2. sudo apt-get install postgresql-client
+   3. sudo apt-get install postgresql postgresql-contrib
```

Comment:
@ngillux Lines 88 to 92 about PostgreSQL can be moved into
https://github.com/freeCodeCamp/classroom/blob/main/.devcontainer/postCreateCommand.sh
as I believe you intend to install the PostgreSQL client.

Reply:
@lloydchang thanks for letting me know! I was curious about that, that clarification helps.

Reply:
@ngillux I'll look at this in your repository and create a pull request for you to review for:
```sudo apt update
sudo apt-get install postgresql-client
```

I am unclear why an additional PostgreSQL server is needed here via
```
sudo apt-get install postgresql postgresql-contrib
```

There is already a PostgreSQL container running on TCP 5432 via https://github.com/freeCodeCamp/classroom/blob/main/.devcontainer/docker-compose.yml#L31-L39
and https://github.com/freeCodeCamp/classroom/blob/main/.devcontainer/devcontainer.json#L36-L38

Reply:
@ngillux Sorry about that.  I empathize and I now see where the confusion came from.

./codespace-instructions.sh says:
> 4.) Run: sudo service postgresql start

That step is extraneous because a PostgreSQL container is now available via
https://github.com/freeCodeCamp/classroom/blob/main/.devcontainer/docker-compose.yml#L31-L39
```
  db:
    image: postgres:latest
    restart: unless-stopped
    volumes:
      - postgres-data:/var/lib/postgresql/data
    environment:
      POSTGRES_PASSWORD: postgres
      POSTGRES_USER: postgres
      POSTGRES_DB: postgres
```
and
https://github.com/freeCodeCamp/classroom/blob/main/.devcontainer/devcontainer.json#L36-L38
```
	// Use 'forwardPorts' to make a list of ports inside the container available locally.
	// This can be used to network with other containers or with the host.
	"forwardPorts": [3000, 3001, 5432, 5555],
```

Reply:
@ngillux https://github.com/ngillux/classroom/pull/1 has the additional changes to incorporate into your branch

Reply:
👆 @Code-Victor

Reply:
I'm going to go ahead and add the setup instructions recommended by @ngillux.  If the postgres setup instructions turn out to be unnecessary, we can remove those instructions later.

Reply:
Relates to https://github.com/freeCodeCamp/classroom/pull/396